src/style.css

```diff
@@ -0,0 +1,82 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: "Book";
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: "Heavy";
+  font-weight: 800;
+}
+
+body {
+  font-family: "Avenir", Arial, Helvetica, sans-serif;
```

Comment:
it's not working for input now

add this style to input, it should fix tests

```
font-family: inherit

Reply:
also remove default outlines
<img width="310" alt="image" src="https://user-images.githubusercontent.com/102411130/224636959-3738d342-6a02-4a84-9f03-273707d0b3fc.png">

for input
```
outline: none